### PR TITLE
fix(login page): removes ariaLabel from checkbox

### LINKED ISSUE #1737: LoginPage - remove aria-label from checkbox
The checkbox in the LoginPage component includes an aria-label in addition to a traditional `<label>`. This attribute should be removed, since we should be using the string defined inside `<label>` only.

### PR DESCRIPTION
**Pull request by jenny-s51**
closes #1737

### CODE REVIEW

**Comment by jgiardino** [comment]
These updates look good! The only question I have is whether this would be a breaking change (and if so, are there any changes we can make now without waiting for a breaking change release)? I'm not able to test what happens if `rememberMeAriaLabel` is defined on `<LoginForm>` (for some reason, the code editor section is read-only).

**Comment by jenny-s51** [comment]
@jgiardino thank you for your feedback! I'm curious as well -- @tlabaj would this be a breaking change?

File: packages/patternfly-4/react-core/src/components/LoginPage/LoginForm.tsx [hunk 1]
```diff
@@ -97,7 +97,6 @@ export const LoginForm: React.FunctionComponent<LoginFormProps> = ({
           label={rememberMeLabel}
           checked={isRememberMeChecked}
           onChange={onChangeRememberMe}
-          aria-label={rememberMeAriaLabel || rememberMeLabel}
```
Comment (dgutride) [inline packages/patternfly-4/react-core/src/components/LoginPage/LoginForm.tsx]: This looks good to me - let's add one more comment up above in rememberMeAriaLabel (to the comment section noting this propType is now deprecated and is no longer used in the code.  Also, please open an issue with a breaking change label indicating we should delete this propType in a future breaking change release.

Comment (jenny-s51) [reply packages/patternfly-4/react-core/src/components/LoginPage/LoginForm.tsx]: Just pushed again with this addition to the comment. Should be all set now -- made a new breaking change issue as well.

**Comment by tlabaj** [comment]
Can you mention new issue here please?  Thnaks.

**Comment by jenny-s51** [comment]
@tlabaj It's #2464 🙂

**Pull request merged**